src/modules/capabilities.json

```diff
@@ -0,0 +1,704 @@
+                    "params": [
+                        {
+                            "name": "capability",
+                            "value": "xrn:firebolt:capability:localization:postalCode"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": true
+                    }
+                }
+            ]
+        },
+        {
+            "name": "info",
+            "summary": "Returns an array of CapabilityInfo objects for the passed in capabilities.",
+            "params": [
+                {
+                    "name": "role",
+                    "required": true,
+                    "schema": {
+                        "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/Role"
+                    }
+                },
+                {
+                    "name": "capabilities",
+                    "required": true,
+                    "schema": {
+                        "type": "array",
+                        "items": {
+                            "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/Capability"
+                        },
+                        "minItems": 1
+                    }
+                }
+            ],
+            "result": {
+                "name": "info",
+                "summary": "Returns an array of CapabilityInfo objects for the passed in capabilities.",
+                "schema": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/CapabilityInfo"
+                    },
+                    "minItems": 1
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default result",
+                    "params": [
+                        {
+                            "name": "role",
+                            "value": "use"
+                        },
+                        {
+                            "name": "capabilities",
+                            "value": [
+                                "xrn:firebolt:capability:device:model",
+                                "xrn:firebolt:capability:keyboard",
+                                "xrn:firebolt:capability:protocol:bluetoothle",
+                                "xrn:firebolt:capability:token:device",
+                                "xrn:firebolt:capability:token:platform",
+                                "xrn:firebolt:capability:protocol:moca",
+                                "xrn:firebolt:capability:wifi:scan",
+                                "xrn:firebolt:capability:localization:postalCode",
+                                "xrn:firebolt:capability:localization:locality"
+                            ]
+                        }
+                    ],
+                    "result": {
```

Comment:
This can maybe come later, but I wonder if we should return more information about the grant? Like it's lifetime on when it will expire?

Reply:
I think this can come later... I'm going to approve this PR soon, so speak up :)